File: app/src/main/java/com/animation/master/AppNavigator.java
package com.animation.master;

import android.content.Context;
import android.content.Intent;
import com.animation.master.dashboard.MainActivity;
import com.animation.master.translationScaleAnimation.TranslationScaleActivity;

public class AppNavigator {

    public static void navigateToDashboard(Context context) {
        Intent intent = new Intent(context, MainActivity.class);
        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
        context.startActivity(intent);
    }

    public static void navigateToTranslationScaleActivity(Context context) {
        Intent intent = new Intent(context, TranslationScaleActivity.class);
        context.startActivity(intent);
    }

}

File: app/src/main/java/com/animation/master/AppNavigator.kt
package com.animation.master

import android.content.Context
import android.content.Intent
import com.animation.master.dashboard.MainActivity
import com.animation.master.moveViewAnimation.MoveViewActivity
import com.animation.master.translationScaleAnimation.TranslationScaleActivity

object AppNavigator {

    fun navigateToDashboard(context: Context) {
        val intent = Intent(context, MainActivity::class.java)
        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK)
        context.startActivity(intent)
    }

    fun navigateToTranslationScaleActivity(context: Context) {
        val intent = Intent(context, TranslationScaleActivity::class.java)
        context.startActivity(intent)
    }

    fun navigateToMoveViewActivity(context: Context) {
        val intent = Intent(context, MoveViewActivity::class.java)
        context.startActivity(intent)
    }

}
